### ISSUE TITLE
Docker Base Image

### ISSUE THREAD

**Issue by eckdanny-osi** [ca-cwds/case-management#22]
Platform front-end teams each seem to be maintaining their own image for running their `rails + react` app. There are common elements (and some diffs). 

### Proposal:

Consolidate as a program. This should:

 - Reduce debugging cycles & integration pain (uniform infra)
 - Improve developer mobility
 - Reduce maintenance
 - Faster updates/upgrades

**Comment by eckdanny-osi** [comment]
@edgibbs @brunoOSI @ratneshraval

**Comment by ratneshraval** [comment]
Good idea. CALS has a pending story in devops backlog to create this image for us.
But it will be nice to have one base for everyone.
